screens/SignInScreen.js

```diff
@@ -1,128 +1,165 @@
-        }
-      >
-        Log In{" "}
-      </Button>
-      <Button
-        onPress={
-          (() => this.SignUp(this.state.email, this.state.password), goToSignUp)
-        }
-      >
-        Don't have an account? Sign up{" "}
-      </Button>
+    <View style={styles.background}>
+      <View style={styles.main}>
+        <View>
+          <View style={styles.logoContainer}>
+            <Image
+              style={styles.logo}
+              source={require("../assets/images/logo.png")}
+            />
+            <Image
+              style={styles.logoText}
+              source={require("../assets/images/wordmark.png")}
+            />
+            {/* Chose not to use because of padding issue
+          <KitText
+            style={styles.logoText}
+            color={Colors.KIT_BLACK}
+            fontWeight="bold"
+            size={100}
+          >
+            kit
+          </KitText> */}
+          </View>
+          <KitText color={Colors.KIT_DARK_GREY} fontWeight="semibold" size={20}>
+            keep in touch
+          </KitText>
+        </View>
+        <View style={styles.textContainer}>
+          <KitText
+            style={styles.text}
+            color={Colors.KIT_BLACK}
+            size={32}
+            fontWeight="bold"
+          >
+            Welcome back!
+          </KitText>
+          <KitText
+            style={styles.text}
+            color={Colors.KIT_LIGHT_ORANGE}
+            size={16}
+            fontWeight="semibold"
+          >
+            Please sign in below.
+          </KitText>
+        </View>
+        <View style={styles.textInputContainer}>
+          <View>
+            <KitTextInput
+              image={require("../assets/images/onboardingemail.png")}
+              placeholder="Email"
+              onChangeText={setEmail}
+              value={email}
+            />
+            <KitTextInput
+              image={require("../assets/images/onboardingpassword.png")}
+              placeholder="Password"
+              onChangeText={setPassword}
+              value={password}
+              secureTextEntry
+            />
+            <KitText color={Colors.KIT_RED} size={16}>
+              {error}
+            </KitText>
+          </View>
+          <KitButtonSupreme
+            onPress={() => {
+              signIn(email, password, setError);
+            }}
+            color={Colors.KIT_LIGHT_ORANGE}
+            width={233}
+          >
+            SIGN IN
+          </KitButtonSupreme>
+          <TouchableOpacity onPress={goToSignUp}>
+            <KitText color={Colors.KIT_RED} fontWeight="semibold" size={16}>
+              Forgot Password?
+            </KitText>
+          </TouchableOpacity>
+        </View>
+        <View style={styles.bottomText}>
+          <KitText color={Colors.KIT_DARK_GREY} fontWeight="medium" size={16}>
+            Don't have an account?{" "}
+          </KitText>
+          <TouchableOpacity onPress={goToSignUp}>
```

Comment:
I'm of the opinion that the only buttons that should be KitButtonSupreme's are the ones that are rounded and colored. Otherwise they should just be text wrapped by a TouchableOpacity.

Reply:
Added a comment in the KitButtonSupreme file